tests/end-to-end/backup-restore-test/restore_cluster_backup_test.go

```diff
@@ -40,59 +35,59 @@ func init() {
 }
 
 func TestBackupAndRestoreCluster(t *testing.T) {
-	myFunctionTest, err := NewFunctionTest()
```

Comment:
Why you are disabling all tests? From where do you know that your new plugin is not harming the other tests?

Reply:
This is just a draft change only for faster tests running feedback, I will enable them as soon as the plugin is proven to be doing the job.